packages/features/flags/hooks/index.ts

```diff
@@ -0,0 +1,17 @@
+import { trpc } from "@calcom/trpc/react";
+
+// TODO: Make the slugs type safe
+export function useFlagSuspense(slug: string) {
```

Comment:
To be used mostly on `packages/features/flags`. Elsewhere we should use the context on client or the server helpers.